build/tools/amebasmart/bootloader/km4_boot_RELEASE_NOTE.txt

```diff
@@ -1,5 +1,9 @@
 /* == "version" + "Realtek git version" + "compile date" + "compile time" == */
 
+== version a1a0ee7deb_2024/12/31-15:44:33 ==
```

Comment:
Because some product is over in dev, it could be impossible to apply.
If possible, I want to avoid bootloader change. How about printing at amebasmart_reboot_reason.c?

Reply:
Inside the BOOT_FaultHandler, two changes were included. 

This change can be moved to amebasmart_reboot_reason.c.
- Display the value of BKUP_REG2 during the BOOT stage.[BOOT-I] BKUP_REG2 0x0

However, we cannot avoid the following change:
When a BOOT fault occurs, ensure BKUP_REG2 is successfully written with the value 0x1 and display it.

Therefore, the bootloader img needs to check the BKUP_REG2 value.
Unless Samsung side agrees to only print the BKUP_REG2 value in the case of KM0 & KM4 fault only.

Reply:
What is the `boot fault`? Fault in bootloader with some reason?

Reply:
BOOT_FaultHandler
for example memory corruption, corrupted bootloader etc…